### DISCUSSION TITLE
Consider multiple retrospectives during the cycle rather than 3 at the end

### DISCUSSION THREAD

**Discussion by saschagrunert** [kubernetes/sig-release#1696] [Ideas]
The idea is to move the retrospectives from the end of the cycle to the second half of it. This provides faster feedback loops to catch-up on issues and follow-ups. It also enables the team to solve possible problems directly during the cycle rather than handing them over to the next team, which they're probably not part of.

Thoughts on that @kubernetes/release-team-leads ?

**Comment by jeremyrickard** [comment]
I personally like this idea. With 1.23 especially, the retros will fall pretty late into December (considering when people start to be unavailable for holiday vacations). 

Doing them earlier will allow for better course correction and problem solving during the cycle, and might make for more opportunities for RT members to contribute during the cycle and be more engaged.

**Reply by reylejano** [reply]
+1 to having a retro during the cycle specially. This is more important with last release cycle at the end of the year as there are multiple holidays.

**Comment by justaugustus** [comment]
How about immediately as we head into burndown?

**Reply by justaugustus** [reply]
(Reply here to discuss timeline)

**Reply by jrsapi** [reply]
+1 to burndown. The enhancement team's role will be about complete, and if there are any changes for the enhancement team this early start would provide time for them to implement before the next release cycle starts.

**Reply by reylejano** [reply]
+1 to starting at burndown. How about coupling the first burndown meeting as the first retro?
If we have a retro during the 1.23 cycle, this would be Monday, Nov 1. 
If Monday is not preferred, how about the Wed or Fri burndown meeting of the first week of burndown?

**Reply by reylejano** [reply]
On [#sig-release](https://kubernetes.slack.com/archives/C2C40FMNF/p1631824312215600?thread_ts=1631822838.214800&cid=C2C40FMNF) Slack, we will have the first retro of the 1.23 cycle on Wed, Nov 3

**Reply by reylejano** [reply]
PR https://github.com/kubernetes/sig-release/pull/1706 updates the 1.23 cycle calendar and moves the first retro to Wed, Nov 3 in week 11 of the release cycle

**Comment by justaugustus** [comment]
cc: @kubernetes/sig-release-leads

**Comment by palnabarun** [comment]
+1 to this idea!

It will help to implement anything that the team wants to improve in the current cycle itself.

**Reply by puerco** [reply]
Exactly and help folks maneuver mid-cycle in a much more coordinated fashion.

**Comment by jdumars** [comment]
+1 to more iterative retros for releases

When I originally started the release-wide retro (1.4 I believe?), it was a much smaller crew of folks involved. I think more frequent retrospectives will provide actionable feedback for the in-progress cycles. I do think we should continue full-release retros that allow for community and non-SIG input, so we can optimize independently for each.
